docs/sections.md

```diff
@@ -20,7 +20,7 @@ Sections help to maintain vertical rhythm.
 </div>
 
 ```html
-<div class="section">
+<div class="section section--first">
```

Comment:
Sure, but it's easer for a developer to remember `section--first` than to remember `push35--top`. 

But I guess a more important reason for this class is so that top section spacing will have the same value as `$section-spacing`.